doc/integrator/make.rst

```diff
@@ -0,0 +1,83 @@
+.. _integrator_make:
+
+Build the project with make
+===========================
+
+Makefiles
+---------
+
+Usually we have the following makefiles includes:
+``<user>.mk`` -> ``<package>.mk`` -> ``CONST_Makefile.mk``.
+
+The ``CONST_Makefile.mk`` is a huge makefile that maintain by the
+c2cgeoportal developers.
+
+The ``<package>.mk`` contains the project specific things,
+by default if contains:
+
+.. code:: makefile
+
+    ifdef VARS_FILE
+    VARS_FILES += ${VARS_FILE} vars_<package>.yaml
+    else
+    VARS_FILE = vars_<package>.yaml
+    VARS_FILES += ${VARS_FILE} 
+    endif
+
+    include CONST_Makefile
+
+
+And the ``<user>.mk`` contains the user specific things, mainly the
+``INSTANCE_ID``, he should contains something like
+
+.. code:: makefile
+
+    INSTANCE_ID = <user>
+    DEVELOPMENT = TRUE
+
+    include <package>.mk
+
+
+Vars files
+----------
+
+We also have some files that describe the variables of the project:
+``vars_<package>.yaml`` that extends the ``CONST_vars.yaml``.
```

Comment:
The project variables are set in the ``vars_<package>.yaml`` file, which extends the default ``CONST_vars.yaml``.